doc/content/htgr/htr-pm/htr-pm_model.md

```diff
@@ -0,0 +1,491 @@
+# SAM Generic PBR Model
+
+*Contact: Zhiee Jhia Ooi, zooi@anl.gov, Stephen Bajorek, stephen.bajorek@nrc.gov*
+
+*Model link: [HTR-PM SAM Model](https://github.com/idaholab/virtual_test_bed/tree/main/htgr/htr-pm)*
+
+!tag name=SAM HTR-PM Model
+  image=
+  pairs=reactor_type:HTGR
```

Comment:
```suggestion
  description=...
  pairs=reactor_type:HTGR
```
forgot about that one

Reply:
what should I add in 'image'? I don't see this in other models.

Reply:
you dont see it locally, but if you rebase you ll see it s in every other model now

Reply:
you pick a picture, with the path you use for the file in doc/content/media/

Reply:
got it. thanks